### PR TITLE
#14982: Update Unary ops examples

### PR DESCRIPTION
**Pull request by VirdhatchaniKN**
### Ticket
#14982

### Problem description
Update Unary examples docs

### What's changed
Updated Unary examples for the following ops : 

- ttnn.hardsigmoid
- ttnn.hardswish
- ttnn.selu

### Checklist
- [ ] Post commit CI

### Doc screenshots
Screenshot 2024-11-21 at 10 04 39 PM
Screenshot 2024-11-21 at 10 05 09 PM
Screenshot 2024-11-21 at 10 05 36 PM

 hardsigmoid sweep test : 
Screenshot 2024-11-21 at 9 34 00 PM
hardswish sweep test : 
Screenshot 2024-11-21 at 9 36 49 PM

### CODE REVIEW

File: tests/sweep_framework/sweeps/eltwise/unary/hardsigmoid/hardsigmoid.py [hunk 1]
```diff
@@ -24,7 +24,7 @@
         + gen_shapes([1, 32, 32], [12, 256, 256], [1, 32, 32], 16)
         + gen_shapes([32, 32], [256, 256], [32, 32], 16),
         "input_a_dtype": [ttnn.bfloat16, ttnn.bfloat8_b],
-        "input_a_layout": [ttnn.TILE_LAYOUT, ttnn.ROW_MAJOR_LAYOUT],
+        "input_a_layout": [ttnn.TILE_LAYOUT],
```
[the remark this answers is not in the document]
Comment (VirdhatchaniKN) [reply tests/sweep_framework/sweeps/eltwise/unary/hardsigmoid/hardsigmoid.py:27]: ok

**Review by Aswinmcw — APPROVED** [review]
LGTM

**Review by mcw-anasuya — APPROVED** [review]
Add Run Sweeps passing link in description

**Review by eyonland — APPROVED** [review]

**Review by KalaivaniMCW — APPROVED** [review]

**Review by mouliraj-mcw — APPROVED** [review]

**Comment by VirdhatchaniKN** [comment]
https://github.com/tenstorrent/tt-metal/actions/runs/11993864842 - passed

**Pull request merged**